apps/server/src/shared/domain/entity/import-user.entity.ts

```diff
@@ -31,7 +31,7 @@ export enum MatchCreator {
 }
 
 @Entity({ tableName: 'importusers' })
-@Unique({ properties: ['school', 'ldapId'] })
+@Unique({ properties: ['school', 'externalId'] })
 @Unique({ properties: ['school', 'ldapDn'] })
 @Unique({ properties: ['school', 'email'] })
 export class ImportUser extends BaseEntityWithTimestamps implements IEntityWithSchool {
```

Comment:
Is that the only UserEntity where you have to rename this?

Reply:
user entity has already been renamed